markets/perps-market/contracts/storage/PerpsAccount.sol

```diff
@@ -516,6 +516,7 @@ library PerpsAccount {
             uint128 amountToLiquidate,
             int128 newPositionSize,
             int128 sizeDelta,
+            uint128 positionFlaggedAmount,
```

Comment:
maybe just return `oldPositionSize`?  `positionFlaggedAmount` is probably unnecessary logic here; can move this to where liquidate is called

Reply:
will maybe rename to something has better meaning here, but the idea is to use that this value is the `abs(oldPositionSize)`, so maybe just that name makes sense: `oldPositionSizeAbs`